importer/likutim/create_unit.go

```diff
@@ -0,0 +1,237 @@
+package likutim
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/pkg/errors"
+	"github.com/spf13/viper"
+	"github.com/volatiletech/sqlboiler/boil"
+	"github.com/volatiletech/sqlboiler/queries/qm"
+	"gopkg.in/volatiletech/null.v6"
+
+	"github.com/Bnei-Baruch/mdb/common"
+	"github.com/Bnei-Baruch/mdb/models"
+	"github.com/Bnei-Baruch/mdb/utils"
+)
+
+type CreateUnits struct {
+	duplicates []*Double
+	mdb        *sql.DB
+}
+
+func (c *CreateUnits) Run() {
+	err := c.duplicatesFromJSON()
+	if err != nil {
+		log.Errorf("Error on read. Error: %s", err)
+		compare := new(Compare)
```

Comment:
Continue after the error is not really intuitive. Maybe add some logging message that says this error is not crucial and you're comparing again.